go/oasis-test-runner/scenario/e2e/runtime/sentry.go

```diff
@@ -130,3 +157,246 @@ func (s *sentryImpl) Fixture() (*oasis.NetworkFixture, error) {
 
 	return f, nil
 }
+
+func (s *sentryImpl) dial(address string, clientOpts *cmnGrpc.ClientOptions, skipClientAuth bool) (*grpc.ClientConn, error) {
+	var creds credentials.TransportCredentials
+	var err error
+	switch skipClientAuth {
+	case false:
+		creds, err = cmnGrpc.NewClientCreds(clientOpts)
+	case true:
+		creds, err = advancedtls.NewClientCreds(&advancedtls.ClientOptions{
+			Certificates:         clientOpts.Certificates,
+			GetClientCertificate: clientOpts.GetClientCertificate,
+			VType:                advancedtls.SkipVerification,
+			VerifyPeer: func(params *advancedtls.VerificationFuncParams) (*advancedtls.VerificationResults, error) {
+				return &advancedtls.VerificationResults{}, nil
+			},
+		})
+	}
+	if err != nil {
+		return nil, err
+	}
+	opts := grpc.WithTransportCredentials(creds)
+	conn, err := cmnGrpc.Dial(address, opts) // nolint: staticcheck
+	if err != nil {
+		return nil, err
+	}
+	return conn, nil
+}
+
+func (s *sentryImpl) loadNodeIdentity(dataDir string) (*identity.Identity, error) {
```

Comment:
Is this something that would be useful to have as a method on nodes?

Reply:
Indeed, moved it there